### ISSUE TITLE
Question: file/dir path of backup_dirs.sh

### ISSUE THREAD

**Issue by MiMaMuh** [nystudio107/craft-scripts#23]
Hi,

thank you for this awesome scripts. I tested your backup_dirs.sh and I was wondering, why the `BACKUP_FILES_DIR_PATH` is not using the same structure like `backup_db.sh` and `backups_assets.sh` including the db name in the path with `${LOCAL_DB_NAME}`:

```sh
backup_dirs.sh:
BACKUP_FILES_DIR_PATH="${LOCAL_BACKUPS_PATH}${FILES_BACKUP_SUBDIR}/"

vs. backup_db.sh and backup_assets.sh
BACKUP_DB_DIR_PATH="${LOCAL_BACKUPS_PATH}${LOCAL_DB_NAME}/${DB_BACKUP_SUBDIR}/"
```

Is there a reason behind the decision?

**Comment by khalwat** [comment]
hmmm it probably should be, I think

**Comment by khalwat** [comment]
Fixed in 1.2.5:

https://github.com/nystudio107/craft-scripts/releases/tag/1.2.5
